build-tools/packages/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -585,6 +585,46 @@ export const handlers: Handler[] = [
             return { resolved: resolved };
         },
     },
+    {
+        name: "npm-package-json-prettier",
+        match,
+        handler: (file) => {
+            let json;
+
+            try {
+                json = JSON.parse(readFile(file));
+            } catch (err) {
+                return "Error parsing JSON file: " + file;
+            }
+
+            const hasPrettierProperty = Object.prototype.hasOwnProperty.call(json, "prettier");
+            const hasPrettierFixProperty = Object.prototype.hasOwnProperty.call(
+                json,
+                "prettier:fix",
+            );
+            const hasFormatProperty = Object.prototype.hasOwnProperty.call(json, "format");
+
+            if (!(hasPrettierProperty && hasPrettierFixProperty && hasFormatProperty)) {
```

Comment:
I think you can remove this if-check since you're now checking them individually.

Reply:
Then would I have to return error if the length of `missingScripts` is greater or equal to 1?

Reply:
>0, I think.

Reply:
Oh, that's the same, I guess. Ignore me. 🤦